layouts/partials/captivate/main_channel.html

```diff
@@ -1,6 +1,21 @@
+{{ $channel := index $.Site.Data.channels (printf "x%s" .Params.channelid ) }}
 <main id="main-content" class="main-content">
-    <div class="container">
-        <div class="row">
+    <section class="channel-page-banner" style="background-image: url('/{{.Type}}/images/channel/{{.Params.channelid}}-banner.jpg');">
```

Comment:
the channels banners is stored in static files in hugo files, it just a temporal fix, because i don't have a way to call it from data base and the channel info avariable in hugo files(content/channelName/channelId.md or data/channels/xchannelId.json) don't provide the images urls, this was touched with Ricky and i know this data and configuration will be provided by themes/tvpage/defaults/captivate.json